package.json

```diff
@@ -1,83 +1,85 @@
 {
   "name": "pci-dss-sanitizer",
-  "version": "0.0.0-development",
+  "version": "0.0.0-development.0",
   "description": "Mask bank card info from strings",
-  "main": "lib/index.js",
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint",
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "npm run check-coverage"
-    }
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": "16"
   },
+  "peerDependencies": {},
   "scripts": {
-    "prebuild": "npm run lint && rimraf lib",
-    "lint": "eslint --cache src/**/*.js",
-    "format": "npm run lint -- --fix",
-    "build": "rollup -c",
-    "watch": "rollup -c --watch",
-    "test": "cross-env NODE_ENV=test jest",
+    "start": "tsdx watch",
+    "build": "tsdx build",
+    "test": "npm run test:client && npm run test:src",
+    "test:src": "tsdx test",
+    "test:client": "npm_scripts/test-client.sh",
```

Comment:
I'm receiving this error on mac:
I'll check how to fix this

```
/Users/demian/skedify/pci-dss-sanitizer/client/test/sh/utils/examples.sh: line 3: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
/Users/demian/skedify/pci-dss-sanitizer/client/test/sh/utils/examples.sh: line 6: 783 91n: value too great for base (error token is "91n")
/Users/demian/skedify/pci-dss-sanitizer/client/test/sh/utils/examples.sh: line 7: /Users/demian/skedify/pci-dss-sanitizer/client/test/sh/utils/assert.sh: No such file or director
```

Reply:
Ah yesh, associative arrays don't work on bash `<=v3` which is default on Mac, I'll see what I can do